packages/macro/test/jsx-trans.ts

```diff
@@ -91,29 +93,39 @@ const cases: TestCase[] = [
   },
   {
     name: "Variables are converted to named arguments",
+    stripId: true,
     input: `
         import { Trans } from '@lingui/macro';
         <Trans>Hi {yourName}, my name is {myName}</Trans>;
       `,
     expected: `
         import { Trans } from "@lingui/react";
-        <Trans id={"Hi {yourName}, my name is {myName}"} values={{
-          yourName: yourName,
-          myName: myName,
-        }} />;
+        <Trans 
+            id={"<stripped>"}
```

Comment:
indentations?

Reply:
if you want, you can polish all indentations by yourself, i don't have time for that